README.md

```diff
@@ -1338,6 +1339,26 @@ See the [example-cron.yml](./.github/workflows/example-cron.yml) workflow.
 
 [![cron example](https://github.com/cypress-io/github-action/workflows/example-cron/badge.svg?branch=master)](.github/workflows/example-cron.yml)
 
+### Suppress test summary
+
```

Comment:
I suggest to add a sentence:

"The default test summary can be suppressed by using the parameter `publish-summary` and setting its value to `false`."

to introduce your explanation.